zebra-rpc/src/methods.rs

```diff
@@ -362,3 +459,44 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 ///
 /// Also see the notes for the [`Rpc::get_best_block_hash` method].
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
+
+/// Response to a `getrawtransaction` RPC request.
+///
+/// See the notes for the [`Rpc::get_raw_transaction` method].
+// pub struct GetRawTransaction(pub(super) SerializedTransaction);
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(untagged)]
+pub enum GetRawTransaction {
+    /// The raw transaction, encoded as hex bytes.
+    Raw(#[serde(with = "hex")] Vec<u8>),
```

Comment:
Should we use a `SerializedTransaction` here instead ?

Reply:
The rest of our hex-encoding code is in `zebra-chain`, so we could move the transaction hex code, tests, and dependencies there for consistency.

Reply:
I don't think it's necessary to move the tests, they're fine in `zebra-rpc`.

I'm using some of the block hex serialization code for the database snapshot tests, and I might do the same with transactions. So if we could leave them in `zebra-chain` that would be great.